### ISSUE TITLE
Replicating the Results as Reported in the Paper

### ISSUE THREAD

**Issue by TzeMeng** [forjiuzhou/Spherical-Knowledge-Distillation#4]
Hi there @forjiuzhou, 

The idea of normalising logits is an interesting one. 

However, I am unable to replicate the results using the codes provided in this respo.  I have set the teacher network to be ResNet50 with the student network as ResNet18. 

The other change I made is to change the Pipeline from reading TFRecords to raw data for ImageNet as discussed here https://github.com/NVIDIA/DALI/blob/eb712f593d98afb87ea56700be7cfd83f512a5f8/docs/examples/use_cases/pytorch/resnet50/main.py

All other hyper-parameters are set to match those in the paper.

**Comment by forjiuzhou** [comment]
> Hi there @forjiuzhou,
> 
> The idea of normalising logits is an interesting one.
> 
> However, I am unable to replicate the results using the codes provided in this respo. I have set the teacher network to be ResNet50 with the student network as ResNet18.
> 
> The other change I made is to change the Pipeline from reading TFRecords to raw data for ImageNet as discussed here https://github.com/NVIDIA/DALI/blob/eb712f593d98afb87ea56700be7cfd83f512a5f8/docs/examples/use_cases/pytorch/resnet50/main.py
> 
> All other hyper-parameters are set to match those in the paper.

One thing to be noticed is to apply Cross Entropy loss after the normalized logits. This should replicate the result here.

Let me know if there is other problems.
